packages/ui-components/src/Input/helpers.tsx

```diff
@@ -1,12 +1,12 @@
 import React from "react";
 import classNames from "classnames/bind";
 import styles from "./styles.module.scss";
-import { InputProps } from "./BaseInput";
+import { CommonInputProps } from "./CommonInput";
```

Comment:
This file is only used for NumberInput, referring to the NumberInput that was already in ui-components when I started this proj. However, I created another NumberInput by changing the type of TextTypeInput, and the prop type (from string to number) -- the new one is a simpler component, easier to understand. I've tested it a little, seems to work as expected.

 I've kept the old NumberInput (and thus this helpers file) incase the new one doesn't work for the use cases it's needed for.